projects/js-packages/publicize-components/src/components/social-previews/modal.js

```diff
@@ -58,6 +70,12 @@ const SocialPreviewsModal = function SocialPreviewsModal( {
 							image={ image }
 							isTweetStorm={ isTweetStorm }
 							tweets={ tweets }
+							connection={ connections?.find( conn => conn.service_name === tab.name ) }
+							excerpt={ excerpt }
+							content={ content }
+							message={ message }
+							mediaDetails={ mediaDetails }
+							attachedMediaDetails={ attachedMediaDetails }
```

Comment:
If we create a separate component here for facebook and use it in constants, we can move these props to that component.